sdk/tests/test_utils/cassettes/test_working_server/test_can_create_process.yaml

```diff
@@ -0,0 +1,122 @@
+        {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{\"content\":\".\"},\"logprobs\":null,\"finish_reason\":null}]}\n\ndata:
+        {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{\"content\":\"
+        How\"},\"logprobs\":null,\"finish_reason\":null}]}\n\ndata: {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{\"content\":\"
+        can\"},\"logprobs\":null,\"finish_reason\":null}]}\n\ndata: {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{\"content\":\"
+        I\"},\"logprobs\":null,\"finish_reason\":null}]}\n\ndata: {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{\"content\":\"
+        assist\"},\"logprobs\":null,\"finish_reason\":null}]}\n\ndata: {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{\"content\":\"
+        you\"},\"logprobs\":null,\"finish_reason\":null}]}\n\ndata: {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{\"content\":\"
+        further\"},\"logprobs\":null,\"finish_reason\":null}]}\n\ndata: {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{\"content\":\"?\"},\"logprobs\":null,\"finish_reason\":null}]}\n\ndata:
+        {\"id\":\"chatcmpl-A156GwD7uq6pHLrv1ZMufdoLPcqOt\",\"object\":\"chat.completion.chunk\",\"created\":1724821296,\"model\":\"gpt-4-turbo-2024-04-09\",\"system_fingerprint\":\"fp_6f7679d977\",\"choices\":[{\"index\":0,\"delta\":{},\"logprobs\":null,\"finish_reason\":\"stop\"}]}\n\ndata:
+        [DONE]\n\n"
```

Comment:
Maybe it's a little nit-picky, but does this string deserve its own file? 😆

Reply:
cassetes are auto generated. this is how vcr stores request/response pairs for tests.